robot-command/src/main/java/org/obolibrary/robot/ReasonCommand.java

```diff
@@ -70,6 +70,14 @@ public ReasonCommand() {
         "exclude-external-entities",
         true,
         "if true, do not add an axiom if it is about classes in external ontologies");
+    o.addOption(
+        "t",
+        "exclude-tautologies",
+        true,
+        "if 'structural', exclude axioms matching a hard-coded set of tautological patterns "
+            + "(e.g., subclass of owl:Thing, superclass of owl:Nothing, X subclass of X); if 'all', use the HermiT reasoner "
```

Comment:
Thi seems a little long for the command line description. Maybe we can add a short sentence and keep the extended description in the docs? e.g., "specify tautological pattern to exclude (structural, all, or false)"